### DISCUSSION TITLE
Residence的模块怎么没了

### DISCUSSION THREAD

**Discussion by Illusionika** [QuickShop-Community/QuickShop-Hikari#371] [General]
你弃坑之前那个版本都有啊，就是要在开店要在领地里还要开flag的。

另外现在支持同事开启多个hook模块么？之前那个版本我本来想同时开WorldGuard和Residence模块, WG的用来开公共跳蚤市场,RES的玩家自己开私人商店,结果这2个模块都开了之后互相单独生效造成冲突,哪边都开不了店

**Comment by Ghost-chu** [comment]
目前在外，回家之后看一下这个问题

**Reply by Ghost-chu** [reply]
通常我建议检查一下是不是开了区域限制功能，有的addon存在只允许显式声明的区域建立商店且为默认配置，因此会互相影响

**Comment by Illusionika** [comment]
啊,看来这第二个问题是逻辑问题,那还是之前那样子.

这个话题的主要问题呢,就是Compat-Residence-X.X.X.X.jar以后会有么?我现在之前的重制版用的res来限制开店区域,看了下现在这版本好像没有

Residence是下面这个插件
https://www.spigotmc.org/resources/residence.11480
